run-k8s-external-storage-e2e.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+#
+# Download the Kubernetes test suite, extract it and run the external-storage suite.
+#
+# Requirements:
+# - KUBE_VERSION needs to be set in the environment (format: "v1.18.5")
+
+# exit on failure
+set -e
+
+[ -n "${KUBE_VERSION}" ] || { echo "KUBE_VERSION not set" ; exit 1 ; }
+
+# download and extract the tests
+curl -LO "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-test-linux-amd64.tar.gz"
+tar xzf kubernetes-test-linux-amd64.tar.gz kubernetes/test/bin/ginkgo kubernetes/test/bin/e2e.test
+
+# generate the rbd-driver.yaml that describes the test features
+cat > rbd-driver.yaml << EOF
+ShortName: cephcsi-rbd-test
+StorageClass:
+  FromExistingClassName: csi-rbd-sc
+  #FromFile: examples/rbd/storageclass.yaml
+
+SnapshotClass:
+  # Must be set to enable snapshotting tests
+  FromName: true
+
+DriverInfo:
+  # Internal name of the driver, this is used as a display name in the test case and test objects
+  Name: test.rbd.csi.ceph.com
+
+  # The range of disk size supported by this driver
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
```

Comment:
do we have any maximum supported size for rbd?

Reply:
I have no idea how this influences the tests. It probably is limited by the storage that is available in the cluster. It comes from some example file.